Update pt_BR.po

marcelocripe opened the pull request:
Hello, Dmitry Groshev.

Please accept the Brazilian Portuguese language translation for the mtPaint program.

The file with the latest translation can also be found at this URL:

https://github.com/marcelocripe/mtPaint_pt_BR/blob/main/mtpaint_pt_BR.po

I've revised all the entries, including the commented entries that didn't need to be translated. I've corrected a number of grammatical errors and I hope I haven't overlooked any other mistakes, as well as including prepositions in sentences where necessary and correcting sentences that lacked the nominal and verbal agreement that is characteristic of my language. And finally, I've included the translations of the new sentences from the most recent update of the translation resource.

I converted the file "mtpaint_pt_BR.po" to "mtpaint.mo" and tested the translation in mtPaint version 3.50-9, which is in antiX Linux edition 23, and I was able to view almost all the sentences I translated, in addition to that, I was able to adjust some line breaks and spacing in the Help texts. Some sentences were not displayed with the translation in mtPaint version 3.50-9, because the ".po" file is from mtPaint version 3.50.01. I hope that the translation of all words and sentences will be displayed correctly in mtPaint version 3.50.01. Since I do not know how to compile the programs, I cannot test the translation in mtPaint version 3.50.01.

Please also accept the translation in the “mtpaint.desktop” file:

https://github.com/marcelocripe/mtPaint_pt_BR/blob/main/mtpaint.desktop

The file “mtpaint.desktop” was translated by the antiX Linux community and is in the Robin-antiX repository:

https://gitlab.com/Robin-antiX/antix23-desktop-files/-/blob/main/applications/mtpaint.desktop?ref_type=heads

I take advantage of this merge request to register my sincere thanks for all your volunteer work and the volunteer work of the other people who are involved in mtPaint and the other GNU/Linux programs.

Thank you very much.

marcelocripe
(Original text in Brazilian Portuguese language)

- - - - -

Olá, Dmitry Groshev.

Por favor, aceite a tradução do idioma o Português do Brasil para o programa mtPaint.

O arquivo que possui a tradução mais recente também podem ser encontrado neste URL:

https://github.com/marcelocripe/mtPaint_pt_BR/blob/main/mtpaint_pt_BR.po

Eu revisei todas as entradas, inclusive as entradas que estão comentadas que não precisavam ser traduzidas. Eu corrigi vários erros gramaticais e espero não ter deixado passar nenhum um outro erro desapercebido pelos meus olhos, além de incluir as preposições nas frases onde é necessário, corrigi as frases que não possuíam a concordância nominal e verbal que é característico no meu idioma. E por fim, incluí as traduções das novas frases da atualização mais recente do recurso de tradução.

Eu converti o arquivo "mtpaint_pt_BR.po" para "mtpaint.mo" e testei a tradução no mtPaint da versão 3.50-9 que está no antiX Linux da edição 23 e consegui visualizar quase todas as frases que traduzi, além de que, consegui ajustar algumas quebras de linhas e os espaçamentos dos textos da Ajuda. Algumas frases não foram exibidas com a tradução na versão 3.50-9 do mtPaint, porque o arquivo ".po" é da versão 3.50.01 do mtPaint. Eu espero que a tradução de todas as palavras e frases sejam exibidas corretamente na versão 3.50.01 do mtPaint. Como eu não sei compilar os programas, eu não tenho como testar a tradução na versão 3.50.01 do mtPaint.

Por favor, aceite também a tradução no arquivo "mtpaint.desktop":

https://github.com/marcelocripe/mtPaint_pt_BR/blob/main/mtpaint.desktop

O arquivo "mtpaint.desktop" foi traduzido pela comunidade do antiX Linux e está no repositório do Robin-antiX:

https://gitlab.com/Robin-antiX/antix23-desktop-files/-/blob/main/applications/mtpaint.desktop?ref_type=heads

Eu aproveito este pedido de mesclagem para registrar os meus sinceros agradecimentos por todo o seu trabalho voluntário e pelo trabalho voluntário das outras pessoas que estão envolvidas no mtPaint e nos outros programas do GNU/Linux.

Muito obrigado.

marcelocripe
(Texto original em idioma Português do Brasil)

[mtpaint.desktop.zip](https://github.com/user-attachments/files/19227646/mtpaint.desktop.zip)

marcelocripe committed:
Update pt_BR.po

Hello, Dmitry Groshev.

Please accept the Brazilian Portuguese language translation for the mtPaint program.

The file with the latest translation can also be found at this URL:

https://github.com/marcelocripe/mtPaint_pt_BR/blob/main/mtpaint_pt_BR.po

I've revised all the entries, including the commented entries that didn't need to be translated. I've corrected a number of grammatical errors and I hope I haven't overlooked any other mistakes, as well as including prepositions in sentences where necessary and correcting sentences that lacked the nominal and verbal agreement that is characteristic of my language. And finally, I've included the translations of the new sentences from the most recent update of the translation resource.

I converted the file "mtpaint_pt_BR.po" to "mtpaint.mo" and tested the translation in mtPaint version 3.50-9, which is in antiX Linux edition 23, and I was able to view almost all the sentences I translated, in addition to that, I was able to adjust some line breaks and spacing in the Help texts. Some sentences were not displayed with the translation in mtPaint version 3.50-9, because the ".po" file is from mtPaint version 3.50.01. I hope that the translation of all words and sentences will be displayed correctly in mtPaint version 3.50.01. Since I do not know how to compile the programs, I cannot test the translation in mtPaint version 3.50.01.

Please also accept the translation in the “mtpaint.desktop” file:

https://github.com/marcelocripe/mtPaint_pt_BR/blob/main/mtpaint.desktop

The file “mtpaint.desktop” was translated by the antiX Linux community and is in the Robin-antiX repository:

https://gitlab.com/Robin-antiX/antix23-desktop-files/-/blob/main/applications/mtpaint.desktop?ref_type=heads

I take advantage of this merge request to register my sincere thanks for all your volunteer work and the volunteer work of the other people who are involved in mtPaint and the other GNU/Linux programs.

Thank you very much.

marcelocripe
(Original text in Brazilian Portuguese language)

- - - - -

Olá, Dmitry Groshev.

Por favor, aceite a tradução do idioma o Português do Brasil para o programa mtPaint.

O arquivo que possui a tradução mais recente também podem ser encontrado neste URL:

https://github.com/marcelocripe/mtPaint_pt_BR/blob/main/mtpaint_pt_BR.po

Eu revisei todas as entradas, inclusive as entradas que estão comentadas que não precisavam ser traduzidas. Eu corrigi vários erros gramaticais e espero não ter deixado passar nenhum um outro erro desapercebido pelos meus olhos, além de incluir as preposições nas frases onde é necessário, corrigi as frases que não possuíam a concordância nominal e verbal que é característico no meu idioma. E por fim, incluí as traduções das novas frases da atualização mais recente do recurso de tradução.

Eu converti o arquivo "mtpaint_pt_BR.po" para "mtpaint.mo" e testei a tradução no mtPaint da versão 3.50-9 que está no antiX Linux da edição 23 e consegui visualizar quase todas as frases que traduzi, além de que, consegui ajustar algumas quebras de linhas e os espaçamentos dos textos da Ajuda. Algumas frases não foram exibidas com a tradução na versão 3.50-9 do mtPaint, porque o arquivo ".po" é da versão 3.50.01 do mtPaint. Eu espero que a tradução de todas as palavras e frases sejam exibidas corretamente na versão 3.50.01 do mtPaint. Como eu não sei compilar os programas, eu não tenho como testar a tradução na versão 3.50.01 do mtPaint.

Por favor, aceite também a tradução no arquivo "mtpaint.desktop":

https://github.com/marcelocripe/mtPaint_pt_BR/blob/main/mtpaint.desktop

O arquivo "mtpaint.desktop" foi traduzido pela comunidade do antiX Linux e está no repositório do Robin-antiX:

https://gitlab.com/Robin-antiX/antix23-desktop-files/-/blob/main/applications/mtpaint.desktop?ref_type=heads

Eu aproveito este pedido de mesclagem para registrar os meus sinceros agradecimentos por todo o seu trabalho voluntário e pelo trabalho voluntário das outras pessoas que estão envolvidas no mtPaint e nos outros programas do GNU/Linux.

Muito obrigado.

marcelocripe
(Texto original em idioma Português do Brasil)

wjaguar commented:
Thanks, I will include your translation in the next version.

marcelocripe commented:
> Thanks, I will include your translation in the next version.

wjaguar, thank you very much.

wjaguar, muito obrigado.

wjaguar commented:
> What specific problem do I need to fix? What is the entry that I need to fix?

Were it only a few entries, I would have fixed them myself without bothering you. The problem is, there are dozens.
I have given you examples; me enumerating every single instance when you can now very well see them yourself at a glance, would be my time wasted for no useful end.

> > The problem is, an interface is NOT a literary text and should NOT be translated like one.
> I am aware of this. Are you aware that people don't use programs when they can't understand what is written in them?

Are you aware that I do not get a broken cent from any of those people who use my code, and anyone dissatisfied with my maintainership is very welcome to fork the project and spend THEIR own time on this thankless chore instead?

The thing I **am** aware of is that here in Europe at least, everyone below 40 is fluent in English, having learned it at school from a tender age. Translations, these days, are a courtesy, not a necessity.

> > String length MATTERS here, first and foremost.
> Which sentence or sentences have a character limit that the volunteer translators need to be alerted to? I look at the ".po" file and I don't see any "#." or "#" entries containing the programmer's comments to the volunteer translators about some kind of space or character limit.

Let this attempt at e-lawyering be your last one in relation to this project.
The "Use Your Common Sense" maxim should be the default for anyone dealing with FOSS, no one is obligated to waste their free time spelling it out for those who want to be difficult.

> > When you translate a short "Delay" into a wordy "Tempo de Atraso"
> Did I misread the word "Delay"? Isn't it a "Tempo de Atraso"? So what kind of "delay" does the text mean? I can make the correction or improvement if you want.

If you really do not understand the difference between the bare noun "Delay" and the noun phrase "Delay time", then it is too soon for you to attempt translating real stuff. Simple as that.
Technical translation is a complicated job, not a game for petulant children. I am maintaining the Russian translation for my project, so I know all the gotchas intimately. You **have** to be as terse as possible, because English is terser than anything non-CJK and the interface elements are laid out for English; you **have** to follow established terminology (such as used in GIMP, Photoshop, and image processing publications) instead of inventing your own on the fly; you **have** to look up the context instead of using the first meaning that comes to mind; you **have** to work around the cases where same English string happens to be used for different things in different places (the spaces at the end of some mtPaint's strings are there to differentiate the uses, where it was impractical to conflate them).
If you do not do those things yourself, is it because you believe someone less valuable than you is obligated to put your toys in order after you done playing? Or what?

> People don't know what "Laplace" or "Roberts" is, and what it's for.

Once again, an UI is not a literary work, we are not at school, no one is obligated to know the history of the tools, and for those curious, there is the Internet.
In an UI the names serve to differentiate the settings, and trial and error serves to direct the choice.

> And how can I guess that in the window title bar or in another part of the window body it already says that "Laplace" is a "filter" or that "Roberts" is an "edge detector"?

How about running the program and looking?

> As I said before, I can't find any "#." or "#" entries containing the programmer's comments to the translators.

For about 20 years, that did not hamper anyone. Do not be a prima donna.

> In my language, it's more important to

NOT. A. LITERARY. WORK.

Do not forget there are loads of programs in a Slackware install that do have a pt and/or pt_BR translation. Do not waste time for both of us playing pretend where I can (and did) look and compare.

>Besides, are you saying that all people on planet Earth are able to read the math symbols "<=" and still understand them?

The people who are using an **image editor**, absolutely should be.

> > Also, the "headline style" of English where most articles are omitted, is used for the very same reason;
> This style of writing in English is a mistake.

It is not your place to decide.
The style exists, and is commonly used in some contexts, such as headlines and software UIs. From the plethora of Portuguese translations of the things installed on my system, I see a similar style used in them as well, have you not seen the same?

The place for you to tell the world it is run wrong, even if you intensely believe it absolutelly is, is NOT the tiny project I am singlehandedly maintaining in my limited free time. If you want to help, you should follow established practices, even if they dissatisfy you for some reason.

This is all. The rest I read, but the wrong reasons do not become less wrong from being reiterated, and all the answers are already given above.

marcelocripe commented:
> Were it only a few entries, I would have fixed them myself without bothering you.

You don't bother me. It's me who doesn't want to bother you or waste your time.

> Are you aware that I do not get a broken cent from any of those people who use my code, and anyone dissatisfied with
> my maintainership is very welcome to fork the project and spend THEIR own time on this thankless chore instead?

I know that, we are all volunteers and we don't earn a single cent for the work we do for the good of hundreds of thousands of people.

> The thing I am aware of is that here in Europe at least, everyone below 40 is fluent in English, having learned it at school from
> a tender age. Translations, these days, are a courtesy, not a necessity.

I can assure you that this is not the reality on the other side of the Atlantic Ocean, to be more specific, in Latin America. Where people have difficulty communicating in their native language and still suffer from the media that forces the use of English words. Many of these words are not used in England or the United States. For example: Notebook, Ultrabook, Netbook to refer to Laptop. I can only read and write in Brazilian Portuguese, I am using the automatic internet translator to communicate with you. I have friends who are fluent in English, but none of them want to donate their time and work doing translations for open source programs.


> How about running the program and looking?

That's exactly what I told you, I tested the translation I made and didn't find any problems, even though the sentences in Brazilian Portuguese are not extremely short as they are in English. I also told you that I couldn't see some sentences in the test I did, because the version of the program I have in antiX 23 (Bookworm) is not the most recent version of mtPaint.

If there are any problems with the sentences that I could not see in the test I did with the file "mtpaint_pt_BR.po", I promise to make as many merge requests as necessary to make it perfect.

Our conversation went down a path that I would not have liked it to have taken. I did not want to take up your time, I just tried to tell you that in each language the number of characters or words increases or decreases in sentences, as in the example I gave above for the sentence "mtPaint Image Editor" and that it is necessary to consider these differences when defining the dimensions of windows, menus, etc. For example, my website https://marcelocripe.eu5.org can be displayed on several different screen sizes because I use percentage values ​​in the CSS code.

I kindly ask that you use the translation I sent, because I tested the translation and it worked in most of the menus and windows in which I viewed the translation. The translation I sent has been corrected and reviewed. The long texts are mostly from the help text, where I was able to adjust the tabs and each of the line breaks that I needed to add in the translation. It will be a waste of material (from the ".po" file I sent you) and time if the most recent "pt_BR" translation is not used.

Please use the translation I sent you.

Thank you very much.

- - - - -

> Were it only a few entries, I would have fixed them myself without bothering you.

Você não me incomoda. Sou eu que não quero te incomodar ou desperdiçar o seu tempo.


> Are you aware that I do not get a broken cent from any of those people who use my code, and anyone dissatisfied with
> my maintainership is very welcome to fork the project and spend THEIR own time on this thankless chore instead?

Eu sei disso, somos todos voluntários e não ganhamos nenhum centavo pelo trabalho que fazemos para o bem de centena de milhares de pessoas.


> The thing I am aware of is that here in Europe at least, everyone below 40 is fluent in English, having learned it at school from
> a tender age. Translations, these days, are a courtesy, not a necessity.

Eu posso garantir para você que esta não é a realidade do outro lado do oceano Atlântico, para ser mais específico, na América Latina. Onde as pessoas possuem dificuldades em se comunicar em seu idioma natal e ainda sofrem com a mídia que força a utilização das palavras em inglês. Várias destas palavras não são utilizadas na Inglaterra ou nos Estados Unidos. Por exemplo: Notebook, Ultrabook, Netbook para se referir ao Laptop. Eu sei ler e escrever apenas em idioma português do Brasil, estou utilizando o tradutor automático da internet para me comunicar com você. Eu tenho amigos que são fluentes no idioma inglês, mas nenhum deles querem doar o seu tempo e trabalho fazendo traduções para os programas de código aberto.


> How about running the program and looking?

Foi exatamente o que eu disse para você, eu testei a tradução que eu fiz e não encontrei nenhum problema, mesmo as frases em idioma português do Brasil não sendo extremamente curtas como ocorre no idioma inglês. Eu também disse para você que algumas frases eu não consegui visualizar no teste que fiz, porque a versão do programa que eu tenho no antiX 23 (Bookworm) não é a versão mais recente do mtPaint.

Se tiver algum problema nas frases que eu não consegui visualizar no teste que fiz com o arquivo "mtpaint_pt_BR.po", eu me comprometo a fazer quantos pedidos de mesclagens forem necessários deixá-lo perfeito.

A nossa conversa seguiu para um caminho no qual eu não gostaria que tivesse seguido. Eu não queria tomar o seu tempo, eu apenas tentei te dizer que em cada idioma a quantidade de caracteres ou palavras aumentam ou diminuem nas frases, conforme o exemplo que citei acima para a frase "mtPaint Image Editor" e que é preciso considerar estas diferenças na hora de definir as dimensões das janelas, dos menus, etc. Por exemplo, a minha página eletrônica https://marcelocripe.eu5.org consegue ser exibida em vários tamanhos de telas diferentes porque utilizo no código CSS valores em porcentagens.

Eu peço encarecidamente que você utilize a tradução que enviei, porque testei a tradução e funcionou na maior parte dos menus e nas janelas que visualizei a tradução. A tradução que enviei está corrigida e revisada. Os textos longos, em sua maioria, são do texto da ajuda, no qual eu consegui ajustar a tabulação e cada uma das quebras de linhas que precisei adicionar na tradução. Será um desperdício de material (do arquivo ".po" que te enviei) e tempo se a tradução "pt_BR" mais recente não for utilizada.

Por favor, utilize a tradução que te enviei.

Muito obrigado.

wjaguar commented:
> suffer from the media that forces the use of English words. Many of these words are not used in England or the United States. For example: Notebook, Ultrabook, Netbook to refer to Laptop.

It is the same in Russian, and moreover, most of terminology is loanwords there. Because Russian words are multisyllable and word formation rules are restrictive, nativized names for anything turn out long and unwieldy unless turned into contractions or acronyms. No one in their right mind wants to enunciate "электронно-вычислительная машина" instead of "компьютер" when talking of a computer, nor "портативный компьютер" (which is two loanwords anyway) instead of "ноутбук" (a calque of "notebook", which to a Russian ear is nicer sounding than "лэптоп", a calque of "laptop").
Human languages are not all equally suited for every use, just as programming languages are not. No amount of wishful thinking can make it not so.

> I can only read and write in Brazilian Portuguese, I am using the automatic internet translator to communicate with you.

Which translator it is? Every one I tried was between bad and laughable when translating to or from Russian or Latvian. Having a better option would be useful.

Still, does that mean that the translation you submitted is an automatic one, which you then checked for grammar mistakes and any obvious nonsense, but could not check for semantic correctness?

Myself, with my native Russian, fluent English, passable Latvian, and basic Spanish, I am able to, at a minimum, notice semantic errors in any European-language translation of mtPaint UI. Thus I do curate every one of them, to spare users from mistranslation hell that I grew to hate.

> If there are any problems with the sentences that I could not see in the test I did with the file "mtpaint_pt_BR.po", I promise to make as many merge requests as necessary to make it perfect.

There are multiple problems with those **messages** (most are NOT sentences!) that you **could** see. This is **the whole point**.

> I kindly ask that you use the translation I sent, because I tested the translation and it worked in most of the menus and windows in which I viewed the translation. The translation I sent has been corrected and reviewed.

The translation is **saying the wrong thing** in a number of places. It is NOT fit to use as is, even for this reason alone.

Given what I now know, I guess a better result could be achieved by us cooperating in the following way: first, **I** use the internet translator you used, to replace the wrong translations by something at least semantically correct; then, **you** fix the inevitable grammar mistakes and bad word choices in the results (my knowledge of the related Spanish will let me catch some of those, but hardly all).

Also, what is your problem with the "headline style"? What I see in Brazilian news headlines, and in the pt_BR translations of GIMP and GTK+, is practically the same thing that is used in English in those contexts. So it obviously is in a wide use.
Ideological hangups do have their place, but when an ideology-driven decision is doing damage, it has to be for a VERY good reason.

marcelocripe commented:
Hello wjaguar.

I will try to respond to each of the paragraphs you wrote.

> It is the same in Russian, and moreover, most of terminology is loanwords there.

I do not speak German, but from talking to other translators and program developers who are native speakers of this language, I believe that the same borrowing process must occur in the German language and the considerable increase in word length.

Look at the comparison between Russian, Brazilian Portuguese and German.

"электронно-вычислительная машина"
"Computador Eletrônico"
"Elektronischer Rechner"

"компьютер"
"Computador"
"Rechner"


> Which translator it is? Every one I tried was between bad and laughable
> when translating to or from Russian or Latvian. Having a better option
> would be useful.

I use Google, DeepL and Leo's automatic translators, but I still need to search the internet when neither of them are enough for me to understand the context of the sentences. In other words, I understand the automatic translation and, based on what I understand, I produce the text of the translation into Brazilian Portuguese.

> Still, does that mean that the translation you submitted is an automatic
> one, which you then checked for grammar mistakes and any obvious
> nonsense, but could not check for semantic correctness?

What you wrote is not true.
I will explain to you what I did.
I found the file "mtpaint_pt_BR.po" with about 70% of the entries translated and with the date of January 20, 2021 as the last update. In my opinion, the Brazilian Portuguese language was abandoned and there was no one maintaining the translation of the mtPaint image editor to “pt_BR”. Did I understand this situation correctly?
Several translations I found in the file "mtpaint_pt_BR.po" were not consistent with each other and in their own texts.

The third entry was not translated:

> #: src/ani.c:654
> msgid "Fix"
> msgstr ""


The text that was in the old translation was:

> #: src/ani.c:759
> msgid "Creating Animation Frames"
> msgstr "Criando Quadros de Animação"

I corrected the text which was an automatic translation to:

> #: src/ani.c:759
> msgid "Creating Animation Frames"
> msgstr "Criando os Quadros da Animação"

The sentence was missing the masculine article “o” agreeing with the plural of the pictures, therefore, the “os” was missing.
Also, the word “animação” is feminine and needs the preposition “de” agreeing with its gender, therefore, the “da” was missing.
We use “de” to indicate something generic, but the text deals with a specific action, therefore, it is not possible to treat it in a generic way.

Note that the sentence below is the same one I found in the old translation, that is, the sentence is perfect and I didn't have to do anything, I just kept it as I found it.

> #: src/ani.c:812
> msgid "Unable to save image"
> msgstr "Não foi possível salvar a imagem"


The same thing happened in the sentence in the entry “#: src/canvas.c:357”, the sentence is perfect and I didn’t have to do anything, I just kept it as I found it.

> #: src/canvas.c:357
> msgid "The image is too large to transform."
> msgstr "A imagem é muito grande para ser transformada."


In the old translation there was this automatic translation:

> #: src/canvas.c:393 src/canvas.c:430 src/otherwindow.c:1613
> #: src/otherwindow.c:2524 src/otherwindow.c:2583 src/otherwindow.c:3179
> msgid "Gradient"
> msgstr "Gradiente"


I corrected it to the term that is much more used, which is:

> #: src/canvas.c:393 src/canvas.c:430 src/otherwindow.c:1613
> #: src/otherwindow.c:2524 src/otherwindow.c:2583 src/otherwindow.c:3179
> msgid "Gradient"
> msgstr "Degradê"

I don't know what the acronym "MT" means. If you could explain its meaning to me, I would be very grateful and I could adapt the translation. In other words, I will replace the acronyms in English with words in Brazilian Portuguese that are understandable to any Brazilian.

> #: src/canvas.c:429
> msgid "MT"
> msgstr "MT"

I could comment on each of the entries that I have modified, corrected or improved, but that would take a long time. If you want to chat with me online (instantly) and with an automatic translator that allows us to communicate easily, use “HexChat”, go to “Libera.Chat” and enter the “antiX-translators” room. Robin-antiX developed the program https://gitlab.com/Robin-antiX/hexchat-encrypt-translatewhich allows us to automatically translate all the sentences that we type and send, with this, several volunteer translators gather in the room and chat (type and read), even if we do not speak a common language. Acronyms are not translated, so never use acronyms in the texts that you will type in HexChat.


> Myself, with my native Russian, fluent English, passable Latvian, and 
> basic Spanish, I am able to, at a minimum, notice semantic errors in
> any European-language translation of mtPaint UI. Thus I do curate every
> one of them, to spare users from mistranslation hell that I grew to hate.

I found several semantic errors in the "mtpaint_pt_BR.po" file, and when I used mtPaint in antiX, I always didn't understand the translations in the program, because they were poorly done. I believe that they were mostly automatic translations, which were copied and pasted, without analyzing the context and how the sentence could be better adapted to make complete sense.


> The translation is saying the wrong thing in a number of places. It is NOT
> fit to use as is, even for this reason alone.

How can you say something you don't understand?

You can check each of the sentences I modified in Google Translate and DeepL, copy from the file "mtpaint_pt_BR.po" and paste into the translator, choose the source language to Brazilian Portuguese and choose the target language to Russian (or English or Latvian or Spanish).

For example, Google refuses to translate the word “Login”, the program believes that the more than 220 million Brazilians know what this word means. The antiX program is called “Login Manager” and I translated/adapted it to “Gerenciador do Início da Sessão”. But it could be “Gerenciador de Autenticação” (this second option would be less understood by the people I asked, with which of the sentences they would be able to understand easily).

Can you understand that I ask some people who do not know the English language which translation they prefer for a certain action that will be performed?

When I explained that they would have to type in their username, press Enter, type in their password, and press Enter, four out of five people preferred the translation “Gerenciador do Início da Sessão”.

> Given what I now know, I guess a better result could be achieved by
> us cooperating in the following way: first, I use the internet translator
> you used, to replace the wrong translations by something at least
> semantically correct; then, you fix the inevitable grammar mistakes
> and bad word choices in the results (my knowledge of the related
> Spanish will let me catch some of those, but hardly all).

I accept your proposal, but we will have to do it in the “HexChat” chat room, in “Libera.Chat” and in the “antiX-translators” room. Because our conversation will be live and instantaneous. We need to agree on a day and time, since we are in different time zones. My time zone is Brasília in Brazil.

I have already translated several programs that Robin-antiX developed with his help, where he explained to me the intention of each sentence, and I was able to obtain a result that was very close to perfection. Because I had the privilege of talking to the creator of the program who guided me through each sentence.

Robin-antiX programs can be found on the page https://gitlab.com/Robin-antiX.


> What I see in Brazilian news headlines, and in the pt_BR translations of GIMP > and GTK+, is practically the same thing that is used in English in those
> contexts.

Gimp is a terrible example of translation into my language, because it is made by people who do not take care of the Brazilian Portuguese language as it deserves. These people do not take into account the need to adapt the translations to make complete sense, and they also do not apply the new orthographic agreement that became mandatory in Brazil on January 1, 2016.

- - - - -

Olá, wjaguar.

Tentarei responder a cada um dos parágrafos que você escreveu.

> It is the same in Russian, and moreover, most of terminology is loanwords there.

Eu não domino o idioma Alemão, mas por conversar com outros tradutores e desenvolvedores de programas que são nativos deste idioma, eu acredito que deve ocorrer o mesmo processo do empréstimo no idioma alemão e o aumento considerável do tamanho das palavras.

Observe a comparação entre o idioma russo, português do Brasil e alemão.

"электронно-вычислительная машина"
"Computador Eletrônico"
"Elektronischer Rechner"

"компьютер"
"Computador"
"Rechner"

> Which translator it is? Every one I tried was between bad and laughable
> when translating to or from Russian or Latvian. Having a better option
> would be useful.

Eu utilizo o tradutor automático do Google, do DeepL e Leo, mesmo assim, eu preciso pesquisar na internet quando nenhum dos dois são suficientes para eu conseguir compreender o contexto das frases. Ou seja, eu compreendo a tradução automática
e a partir do que compreendi produzo o texto da tradução do idioma português do Brasil.


> Still, does that mean that the translation you submitted is an automatic
> one, which you then checked for grammar mistakes and any obvious
> nonsense, but could not check for semantic correctness?

Isso que você escreveu não é verdade.
Eu explico para você o que eu fiz.
Eu encontrei o arquivo "mtpaint_pt_BR.po" com cerca de 70 % das entradas traduzidas e com a data de 20 de janeiro de 2021 como sendo a última atualização. Ao meu ver, o idioma português do Brasil estava abandonado e não havia ninguém mantendo a tradução do editor de imagens mtPaint para “pt_BR”. Eu compreendi esta situação corretamente?
Várias traduções que encontrei no arquivo "mtpaint_pt_BR.po" não estavam consistentes entre si e em seus próprios textos.

A terceira entrada não estava traduzida:

> #: src/ani.c:654
> msgid "Fix"
> msgstr ""


O texto que estava na tradução antiga era:

> #: src/ani.c:759
> msgid "Creating Animation Frames"
> msgstr "Criando Quadros de Animação"

Eu corrigi o texto que era uma tradução automática para:


> #: src/ani.c:759
> msgid "Creating Animation Frames"
> msgstr "Criando os Quadros da Animação"

Faltavam na frase, o artigo masculino “o” concordando com o plural dos quadros, por tanto, faltava o “os”.
Bem como, a palavra “animação” é feminina e precisa da preposição “de” concordando com o seu gênero, por tanto, faltava o “da”.
Utilizamos o “de” para indicar algo genérico, mas o texto trata-se de uma ação específica, por tanto, não cabe tratar de maneira genérica.


Observe que a frase abaixo é a mesma que eu encontrei na tradução antiga, ou seja, a frase está perfeita e não tive que fazer nada, apenas mantive como eu a encontrei.

> #: src/ani.c:812
> msgid "Unable to save image"
> msgstr "Não foi possível salvar a imagem"


O mesmo ocorreu na frase da entrada “#: src/canvas.c:357”, a frase está perfeita e não tive que fazer nada, apenas mantive como eu a encontrei.

> #: src/canvas.c:357
> msgid "The image is too large to transform."
> msgstr "A imagem é muito grande para ser transformada."


Na tradução antiga tinha esta tradução automática:

> #: src/canvas.c:393 src/canvas.c:430 src/otherwindow.c:1613
> #: src/otherwindow.c:2524 src/otherwindow.c:2583 src/otherwindow.c:3179
> msgid "Gradient"
> msgstr "Gradiente"


Eu corrigi para o termo que é muito mais utilizado que é o:

> #: src/canvas.c:393 src/canvas.c:430 src/otherwindow.c:1613
> #: src/otherwindow.c:2524 src/otherwindow.c:2583 src/otherwindow.c:3179
> msgid "Gradient"
> msgstr "Degradê"

Eu não sei o que significa a sigla “MT”, se você puder me explicar o seu significado, eu ficarei muito grato e poderei fazer a adaptação da tradução. Ou seja, irei substituir as siglas em idioma inglês por palavras em idioma português do Brasil que é compreensível por qualquer brasileiro.

> #: src/canvas.c:429
> msgid "MT"
> msgstr "MT"


Eu poderia comentar sobre cada uma das entradas que modifiquei, corrigi ou melhorei, mas isso iria demorar muito tempo. Se você quiser conversar comigo on-line (de maneira instantânea) e com um tradutor automático que permite nos comunicar facilmente, utilize o “HexChat”, acesse o “Libera.Chat” e entre na sala “antiX-translators”. O Robin-antiX desenvolveu o programa https://gitlab.com/Robin-antiX/hexchat-encrypt-translate, que permite traduzir automaticamente todas as frases que digitamos e enviamos, com isso, vários tradutores voluntários se reúnem na sala e conversam (digitam e leem), mesmo sem falarmos um idioma em comum. As siglas não são traduzidas, por tanto, nunca utilize siglas nos textos que você irá digitar no HexChat.


> Myself, with my native Russian, fluent English, passable Latvian, and 
> basic Spanish, I am able to, at a minimum, notice semantic errors in
> any European-language translation of mtPaint UI. Thus I do curate every
> one of them, to spare users from mistranslation hell that I grew to hate.

Eu encontrei vários erros de semântica no arquivo "mtpaint_pt_BR.po", e quando utilizava o mtPaint no antiX, eu sempre ficava sem compreender as traduções que estavam no programa, porque estavam mal feitas. Eu acredito que eram em sua maioria traduções automáticas, que foram copiadas e coladas, sem uma análise do contexto e de como poderia ser melhor adaptada a frase para fazer sentido completo.


> The translation is saying the wrong thing in a number of places. It is NOT
> fit to use as is, even for this reason alone.

Como você pode afirmar algo que você não compreende?
Você pode verificar cada uma das frases que modifiquei no tradutor do Google e no DeepL, copie do arquivo "mtpaint_pt_BR.po" e cole no tradutor, escolha o idioma de origem para português do Brasil e escolha o idioma de destino para russo (ou inglês ou letão ou espanhol).

Por exemplo, o Google se recusa a traduzir a palavra “Login”, o programa acredita que os mais de 220 milhões de brasileiros sabem o que esta palavra significa. O programa do antiX se chama “Login Manager” e eu o traduzi/adaptei para “Gerenciador do Início da Sessão”. Mas poderia ser “Gerenciador de Autenticação” (esta segunda opção seria menos compreendida pelas pessoas que perguntei, com qual das frases conseguiria entender facilmente).

Você consegue compreender que eu pergunto para algumas pessoas que não conhecem o idioma inglês qual tradução elas preferem para uma determinada ação que será realizada?

Quando eu expliquei que iria ter que digitar o nome de usuário, pressionar a tecla Enter, digitar a senha e pressionar a tecla Enter, quatro de cinco pessoas preferiram a tradução “Gerenciador do Início da Sessão”.


> Given what I now know, I guess a better result could be achieved by
> us cooperating in the following way: first, I use the internet translator
> you used, to replace the wrong translations by something at least
> semantically correct; then, you fix the inevitable grammar mistakes
> and bad word choices in the results (my knowledge of the related
> Spanish will let me catch some of those, but hardly all).

Eu aceito a sua proposta, mas teremos que fazer isso na sala do bate-papo do “HexChat”, no “Libera.Chat” e na sala “antiX-translators”. Porque a nossa conversa será ao vivo e instantânea. Precisamos combinar o dia e o horário, haja vista que estamos em fusos horários diferentes. O meu fuso horário é o de Brasília no Brasil.

Eu já traduzi vários programas que o Robin-antiX desenvolveu com a ajuda dele, onde ele me explicava a intenção de cada frase, e eu consegui obter um resultado muito próximo da perfeição. Porque eu tive o privilégio de conversar com o criador do programa que me orientou em cada frase.
Os programas do Robin-antiX podem ser encontrados na página https://gitlab.com/Robin-antiX.


> What I see in Brazilian news headlines, and in the pt_BR translations of GIMP > and GTK+, is practically the same thing that is used in English in those
> contexts.

O Gimp é um péssimo exemplo de tradução para o meu idioma, porque é feito por pessoas que não cuidam do idioma português do Brasil como ele merece. Estas pessoas não levam em consideração a necessidade de se adaptar as traduções para fazerem sentido completo, e ainda, não aplicam o novo acordo ortográfico que passou a ser obrigatório no Brasil desde o dia 01 de janeiro de 2016.

wjaguar commented:
> I do not speak German, but from talking to other translators and program developers who are native speakers of this language, I believe that the same borrowing process must occur in the German language and the considerable increase in word length.

While Germans do on occasion create overlong compound words for a laugh, I do NOT observe any such BS in real projects' de.po files; neither in GIMP, nor in GTK+, nor in mtPaint. And neither do you, so spare me any empty handwaving.

> "компьютер" "Computador" "Rechner"

Which is why a sizable part of Russian terminology is calques from German; English is not the sole source.
In Latvian we have the same thing; in cases a term can be nativized without becoming an eyesore, the native form is what people use, like "dators" for a computer. When no such luck, loanwords become established in common usage.

> I use Google, DeepL and Leo's automatic translators,

No luck, then. :-(
Google and DeepL both mistranslate, lose, or hallucinate at least one item in every paragraph. Hallucinating nonexistent words in Latvian is also a thing.
The tools may be passable for unimportant mundane stuff where neither mistakes nor the content itself are of any consequence, but they are NOT fit for translating tech or legalese. Definitely not now, and possibly never.

> What you wrote is not true. I will explain to you what I did. I found the file "mtpaint_pt_BR.po" with about 70% of the entries translated and with the date of January 20, 2021 as the last update. In my opinion, the Brazilian Portuguese language was abandoned and there was no one maintaining the translation of the mtPaint image editor to “pt_BR”. Did I understand this situation correctly?

You did. However, a mistranslation is WORSE than none at all, and an illiterate with a halfbaked tech tool is NOT fit to maintain any damn thing.
Were it otherwise, I would have used those same tools to make a "translation" myself. I do not do it because I **tried it** and I was **not** happy with what those things spewed out.

> The third entry was not translated:
> > #: src/ani.c:654
> > msgid "Fix"
> > msgstr ""

And you "translated" it WRONG, dude. It is "fix" as in "place definitely", NOT as in "repair". You FAILED to look up the context, therefore you wasted both your time putting it there, and then my time, for me having to tell you of it.
That is a **net negative**.

> I corrected the text which was an automatic translation to:
...
> The sentence was missing the masculine article “o” agreeing with the plural of the pictures,

The "headline style". Am I right that you violate it for the sole reason that you use an automatic tool **not fit for the job** and are taking its sorry product as God's own truth?
That is a **net negative** yet again.

> Note that the sentence below is the same one I found in the old translation, that is, the sentence is perfect and I didn't have to do anything, I just kept it as I found it.
> > #: src/ani.c:812
> > msgid "Unable to save image"
> > msgstr "Não foi possível salvar a imagem"

The version 3.40 happened long enough ago that I do not remember whether I had been checking translation changes item by item back then. Either I had not, or I accepted this because the Spanish translation had the article too; and **in this specific case** I would not be against seeing a "the" in the English message there either, so I consider its counterpart in a translation an acceptable variation.
Your carpetbombing the translation by machine-generated spam, on the other hand, is NOT acceptable. Where neither the source, nor the existing pt_BR, nor ANY other translation use an article, that is a clear vote of human beings against stupid "AI" software.

> I corrected it to the term that is much more used, which is:
> > #: src/canvas.c:393 src/canvas.c:430 src/otherwindow.c:1613
> > #: src/otherwindow.c:2524 src/otherwindow.c:2583 src/otherwindow.c:3179
> > msgid "Gradient"
> > msgstr "Degradê"

**This** is good; I checked and GIMP uses the same in its pt_BR.

> I don't know what the acronym "MT" means.

It means "Mark Tyler", and should be left untranslated. The acronym is a perfectly good label as it is.

> In other words, I will replace the acronyms in English with words in Brazilian Portuguese that are understandable to any Brazilian.

Are you purposely ignoring anything I say?
I said messages MUST be terse and I MEANT IT. If you have ideas which cannot work with this constraint, take them ELSEWHERE. I am NOT sacrificing interface layout to accommodate ANY useless cruft.

I will not be repeating this again. Any and all verbosity for verbosity's sake WILL be rejected with prejudice.

> I found several semantic errors in the "mtpaint_pt_BR.po" file, and when I used mtPaint in antiX, I always didn't understand the translations in the program, because they were poorly done. I believe that they were mostly automatic translations, which were copied and pasted, without analyzing the context and how the sentence could be better adapted to make complete sense.

Bullshitting me is a VERY bad idea.
Now, either you demonstrate me those "poorly done translations" you "always didn't understand" - that means at least 3 of them which I myself could misunderstand; or you get booted out of here for lying and that will be the end of it. Nothing good can be done in bad faith.

> > The translation is saying the wrong thing in a number of places. It is NOT
> > fit to use as is, even for this reason alone.
> How can you say something you don't understand?

Do not fool yourself. Understanding is easy when one knows a related language, and online dictionaries solve the rest.

> You can check each of the sentences I modified in Google Translate and DeepL

Take them to Google and DeepL then, and stop boring me.
Here, I check the sentences against the translations in other image-related software, primarily GIMP and GTK+. What is good for them with their large dedicated language teams, is good for me too.
The **large software projects** set the standard for how to name what. You FOLLOW that standard. No Google, no DeepL, and certainly not you, is permitted to replace the terminology that is in wide use for decades.

> Can you understand that I ask some people who do not know the English language which translation they prefer for a certain action that will be performed?

You are about 50 years late with your crusade. And I already told you that any crusading is to be done OUT of this project.

> Gimp is a terrible example of translation into my language, because it is made by people who do not take care of the Brazilian Portuguese language as it deserves.

When you start caring for mental constructs and lofty ideas to the detriment to human beings, is when you become a liability for any software project in existence. HUMANS is who deserve our care, and we do that by providing them tools with USABLE interfaces.

marcelocripe commented:
(Следующий текст на русском языке является автоматическим переводом, выполненным Google Translate)

> No luck, then. :-(

Все тексты на английском языке, которые я вам отправлял во время нашего разговора здесь, в этой теме, являются автоматическими переводами, которые Google сделал с моего оригинального текста на бразильском португальском.

Подумайте об этом, если вы можете общаться со мной с помощью автоматических переводов, если хотя бы 90% текста вам понятны, то в каком-то смысле мы можем общаться. Может быть, для меня будет разумнее отправить вам автоматический перевод с русского языка.


> Google and DeepL both mistranslate, lose, or hallucinate at least one item
> in every paragraph.

Вы абсолютно правы по поводу неточностей и ошибок, автоматические переводчики - это программы и они несовершенны. Например, Google часто пропускает слово "Please" при переводе на "pt_BR", я не знаю, почему Google часто не переводит одно из самых важных для человека слов, которое "Por favor".


> And you "translated" it WRONG, dude. It is "fix" as in "place definitely",
> NOT as in "repair". You FAILED to look up the context, therefore you
> wasted both your time putting it there, and then my time, for me
> having to tell you of it. That is a net negative.

Если правильный термин — что-то другое, то почему вы использовали слово «исправить»?

Конечно, другие языки были затронуты негативно. Обратите внимание на перевод языка https://github.com/wjaguar/mtPaint/blob/master/po/es.po.

> #: src/ani.c:654
> msgid "Fix"
> msgstr "Fijar"

Ознакомьтесь с вариантами перевода слова «Fijar», которые предлагает DeepL https://www.deepl.com/pt-BR/translator#es/pt-br/Fijar

"Consertar, Fixar, Corrigir, Conserto"

Давайте посмотрим на другой язык https://github.com/wjaguar/mtPaint/blob/master/po/ru.po

> #: src/ani.c:654
> msgid "Fix"
> msgstr "Зафиксировать"

https://www.deepl.com/pt-BR/translator#ru/pt-br/%D0%97%D0%B0%D1%84%D0%B8%D0%BA%D1%81%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D1%82%D1%8C
"Consertar, Conserte-o, Corrigi-lo, Conserte"

Но вот в чем вопрос: Что следует исправить? Или что следует улучшить?

Может быть, это два-три слова?

Или еще лучше, вы могли бы получить комментарии добровольца-программиста для добровольцев-переводчиков, так же, как это сделал Robin-antiX в программе https://gitlab.com/Robin-antiX/antixscreenshot2/-/blob/main/po/antixscreenshot2.pot?ref_type=heads.


> This is good; I checked and GIMP uses the same in its pt_BR.

Теперь я не понимаю, вы жалуетесь на то, что я сделал, но вы же узнали что-то хорошее?! Это то, что я сделал в каждом из предложений, которые я изменил в файле "pt_BR.po".


> It means "Mark Tyler", and should be left untranslated. The acronym
> is a perfectly good label as it is.

Спасибо за разъяснение значения аббревиатуры «MT», потому что я не знаю, какое отношение эта аббревиатура имеет к программе редактирования изображений.

К какому «Марку Тайлеру» относится программа mtPaint?
Может ли это быть кто-то из этих людей?

https://en.wikipedia.org/wiki/Mark_Tyler_(footballer)
https://pt.wikipedia.org/wiki/Mark_Tyler


> Are you purposely ignoring anything I say?
> I said messages MUST be terse and I MEANT IT. If you have ideas
> which cannot work with this constraint, take them ELSEWHERE. I
> am NOT sacrificing interface layout to accommodate ANY useless
> cruft.

Я не согласен с вами по одной простой причине: ни один человек на планете Земля не обязан знать, что означают аббревиатуры в английском языке. Здесь, в Бразилии, уже есть несколько, которые только сбивают людей с толку (NIS, CNIS, INSS, PMSP, PMG, PMRJ, PF, TJ, TRE, TSE и т. д.). Я согласен с вами, что общение должно быть ясным и объективным, но я полностью не согласен с использованием аббревиатур для общения.


> Attempting to deceive me is a VERY bad idea.
> Now, either you demonstrate me those "poorly done translations"
> you "always didn't understand" - that means at least 3 of them which I
> myself could misunderstand; or you get booted out of here for lying
> and that will be the end of it. Nothing good can be done in bad faith.

Я не пытаюсь никого обмануть. Зачем мне пытаться обмануть вас? Что я получу, если попытаюсь обмануть вас? Я думал, мы добровольцы, вы — добровольный программист, а я — добровольный переводчик.

Поскольку у меня все еще есть старый файл перевода «pt_BR», я могу указать вам на некоторые записи, которые я не смог понять.
Записи, которые не были переведены: #: src/ani.c:654, #: src/ani.c:954, #: src/canvas.c:399, #: src/canvas.c:403, #: src/canvas.c:410, etc.

Записи, которые переведены или не переведены: #: src/canvas.c:429, #: src/canvas.c:429, #: src/canvas.c:430, #: src/canvas.c:430, etc.

Взгляните на старый файл (mtpaint_pt_BR_Original_20-01-2021.po) и обратите внимание, сколько предложений не переведено. Так как же вы ожидаете, что я пойму программу, если она не переведена на мой язык?

Я пожертвовал своим временем и волонтерской работой, чтобы перевести ее наилучшим образом, я старался изо всех сил. Если она не идеальна, любой другой (носитель бразильского португальского) может внести необходимые улучшения. Я уже взял на себя обязательство сделать столько запросов на слияние, сколько необходимо, чтобы получить наилучший возможный перевод. Что я могу сделать, так это использовать файл https://github.com/wjaguar/mtPaint/blob/master/po/ru.po качестве справочного материала, поскольку теперь я знаю, что это ваш родной язык. Я, безусловно, могу получить еще лучший результат, если проверю автоматический перевод Google и DeepL с русского на бразильский португальский. Вот что я могу сделать, и я обязуюсь делать больше запросов на слияние.

Что еще вы хотите от меня?

[mtpaint_pt_BR_Original_20-01-2021_po.zip](https://github.com/user-attachments/files/20853121/mtpaint_pt_BR_Original_20-01-2021_po.zip)

- - - - -

> No luck, then. :-(

Todos os textos em idioma inglês que eu enviei para você durante a nossa conversa aqui neste tópico são traduções automáticas, que foram feitas pelo Google a partir do meu texto original em idioma português do Brasil.

Pense bem, se você está sendo capaz de se comunicar comigo por meio das traduções automáticas, se pelo menos 90% do texto é compreensível para você, então de alguma maneira nós estamos conseguindo nos comunicar. Talvez faça mais sentido eu te enviar a tradução automática do idioma russo.


> Google and DeepL both mistranslate, lose, or hallucinate at least one item
> in every paragraph.

Você tem toda razão quanto as imprecisões e os erros, os tradutores automáticos são programas e não são perfeitos. Por exemplo, o Google costuma omitir na tradução para "pt_BR" da palavra "Please", eu não sei porque o Google costuma não traduzir uma das palavras mais importantes para os seres humanos que é o "Por favor".


> And you "translated" it WRONG, dude. It is "fix" as in "place definitely",
> NOT as in "repair". You FAILED to look up the context, therefore you
> wasted both your time putting it there, and then my time, for me
> having to tell you of it. That is a net negative.

Se o termo correto é outro, então por que você usou a palavra "fix"?

Certamente outros idiomas foram afetados negativamente. Observe a tradução do idioma https://github.com/wjaguar/mtPaint/blob/master/po/es.po.

> #: src/ani.c:654
> msgid "Fix"
> msgstr "Fijar"

Olhe as possibilidades para as traduções da palavra "Fijar" que o DeepL oferece https://www.deepl.com/pt-BR/translator#es/pt-br/Fijar

"Consertar, Fixar, Corrigir, Conserto"

Vamos observar um outro idioma https://github.com/wjaguar/mtPaint/blob/master/po/ru.po

> #: src/ani.c:654
> msgid "Fix"
> msgstr "Зафиксировать"

https://www.deepl.com/pt-BR/translator#ru/pt-br/%D0%97%D0%B0%D1%84%D0%B8%D0%BA%D1%81%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D1%82%D1%8C
"Consertar, Conserte-o, Corrigi-lo, Conserte"

Mas a questão é: O que deve ser corrigido? Ou o que deve ser melhorado?

Seria possível ter duas ou três palavras?

Ou melhor, poderia ter os comentários do programador voluntário para os tradutores voluntários, da mesma maneira que o Robin-antiX fez no programa https://gitlab.com/Robin-antiX/antixscreenshot2/-/blob/main/po/antixscreenshot2.pot?ref_type=heads.


> This is good; I checked and GIMP uses the same in its pt_BR.

Agora eu fiquei sem entender, você reclama do que eu fiz, mas reconheceu algo de bom?! Foi o que eu fiz em cada um das frases que modifiquei no arquivo "pt_BR.po".


> It means "Mark Tyler", and should be left untranslated. The acronym
> is a perfectly good label as it is.

Obrigado por esclarecer o que significa a sigla "MT", porque eu não sei o que esta sigla tem haver com um programa de edição de imagens. 

De qual "Mark Tyler" o programa mtPaint está se referindo?
Pro acaso seria alguma destas pessoas?
https://en.wikipedia.org/wiki/Mark_Tyler_(footballer)
https://pt.wikipedia.org/wiki/Mark_Tyler


> Are you purposely ignoring anything I say?
> I said messages MUST be terse and I MEANT IT. If you have ideas
> which cannot work with this constraint, take them ELSEWHERE. I
> am NOT sacrificing interface layout to accommodate ANY useless
> cruft.

Eu discordo de você, por um simples motivo, porque nenhum ser humano no planeta Terra tem a obrigação de saber o que significam as siglas do idioma inglês. Aqui no Brasil já existem várias que só servem para confundir as pessoas (NIS, CNIS, INSS, PMSP, PMG, PMRJ, PF, TJ, TRE, TSE, etc.). Eu concordo com você que a comunicação tem que ser clara e objetiva, mas discordo totalmente em utilizar siglas para se comunicar.


> Attempting to deceive me is a VERY bad idea.
> Now, either you demonstrate me those "poorly done translations"
> you "always didn't understand" - that means at least 3 of them which I
> myself could misunderstand; or you get booted out of here for lying
> and that will be the end of it. Nothing good can be done in bad faith.

Eu não estou tentando enganar ninguém. Por qual motivo eu iria tentar te enganar? O que eu ganho se eu em tentar te enganar? Eu pensava que fossemos voluntários, você um programador voluntário e eu um tradutor voluntário.

Como eu ainda tenho guardado o arquivo antigo da tradução "pt_BR", eu posso te indicar algumas entradas que eu não conseguia compreender.
As entradas que não estavam traduzidas: #: src/ani.c:654, #: src/ani.c:954, #: src/canvas.c:399, #: src/canvas.c:403, #: src/canvas.c:410, etc.

As entradas que estão ou não traduzidas: #: src/canvas.c:429, #: src/canvas.c:429, #: src/canvas.c:430, #: src/canvas.c:430, etc.

Olhe você mesmo o arquivo antigo (mtpaint_pt_BR_Original_20-01-2021.po) e o observe a quantidade de frases que não estavam traduzidas. Então como você quer que eu compreenda o programa que não está traduzido em meu idioma?

Eu doei o meu tempo e trabalho voluntário para traduzir da melhor maneira possível, eu me esforcei para fazer o meu melhor. Se não ficou perfeito, qualquer outra pessoa (nativa do idioma o português do Brasil) poderá fazer as melhorias necessárias. Eu já me comprometi a fazer quantos pedidos de mesclagens forem necessários para ter a melhor tradução possível. O que eu posso fazer e utilizar o arquivo https://github.com/wjaguar/mtPaint/blob/master/po/ru.po como referência, uma vez que agora eu sei que este é o seu idioma nativo. Certamente poderei obter um resultado ainda melhor se verificar a tradução automática do Google e do DeepL do idioma russo para o português do Brasil. Isso é o que eu posso fazer e me comprometo a fazer outros pedidos de mesclagens.

O que mais você quer de mim?


[mtpaint_pt_BR_Original_20-01-2021_po.zip](https://github.com/user-attachments/files/20853121/mtpaint_pt_BR_Original_20-01-2021_po.zip)

marcelocripe committed:
Update pt_BR.po

marcelocripe committed:
Update pt_BR.po
